### PR TITLE
README.md: add kernel building section

### PR DESCRIPTION
**Pull request by ThibF**
Add kernel building instructions.

### CODE REVIEW

**Comment by ThibF** [comment]
Require https://kernel.ubuntu.com/gitea/kernel/kteam-tools/pulls/696 merged to be working

File: README.md [hunk 1]
```diff
@@ -620,7 +621,41 @@ Here are example instructions for building QEMU (for normal user with sudo right
 	For details, you can refer to https://wiki.debian.org/BuildingTutorial#Installing_and_testing_the_modified_package
 
 
+<a id="build-kernel-from-source"></a>
+## 11. Build Kernel from Source
+
+1. Initialize a matching build environment.
+
+	```bash
```
[the remark this answers is not in the document]
Comment (ThibF) [reply README.md:629]: Indeed, thanks for catching that, it was a tab vs space issue.

File: README.md [hunk 2]
```diff
@@ -620,7 +621,41 @@ Here are example instructions for building QEMU (for normal user with sudo right
...
+	cd <kernel repository>
+	<path-to-kteam-tools>/cranky/cranky fdr clean binary
+	```
+
+4. Install the kernel.
+
+Example of kernel installation:
+	```bash
```
Comment (spmcmillan) [inline README.md:654]: The fourth bullet item is still off in preview mode.  1,2,3 are good.

**Comment by hector-cao** [comment]
@ThibF  Why we could not rely on public documentation (or at least have similar instructions) to build the kernel ?

**Comment by ThibF** [comment]
> @ThibF Why we could not rely on public documentation (or at least have similar instructions) to build the kernel ?

1. So I can be specific to this specific usecase (noble:linux-intel) 
2. Our [current docs](https://canonical-kteam-docs.readthedocs-hosted.com/en/public/index.html) are being updated and I'm working on that, but right now it is not using/avoiding the kteam-tools. I think it's a mistake as our tools are continuously updated to include all edge case and are used daily. Trying to avoid the kteam-tools leads to exception and edge cases for each series (see [here](https://git.launchpad.net/~canonical-kernel/+git/kteam-tools/tree/cranky/cmds.d/chroot#n180))

**Comment by hector-cao** [comment]
> > @ThibF Why we could not rely on public documentation (or at least have similar instructions) to build the kernel ?
> 
> 1. So I can be specific to this specific usecase (noble:linux-intel)
> 2. Our [current docs](https://canonical-kteam-docs.readthedocs-hosted.com/en/public/index.html) are being updated and I'm working on that, but right now it is not using/avoiding the kteam-tools. I think it's a mistake as our tools are continuously updated to include all edge case and are used daily. Trying to avoid the kteam-tools leads to exception and edge cases for each series (see [here](https://git.launchpad.net/~canonical-kernel/+git/kteam-tools/tree/cranky/cmds.d/chroot#n180))

Can we use similar instructions as the ones in https://wiki.ubuntu.com/Kernel/BuildYourOwnKernel to build -intel kernel ?

**Comment by ThibF** [comment]
> > > @ThibF Why we could not rely on public documentation (or at least have similar instructions) to build the kernel ?
> > 
> > 
> > 
> > 1. So I can be specific to this specific usecase (noble:linux-intel)
> > 2. Our [current docs](https://canonical-kteam-docs.readthedocs-hosted.com/en/public/index.html) are being updated and I'm working on that, but right now it is not using/avoiding the kteam-tools. I think it's a mistake as our tools are continuously updated to include all edge case and are used daily. Trying to avoid the kteam-tools leads to exception and edge cases for each series (see [here](https://git.launchpad.net/~canonical-kernel/+git/kteam-tools/tree/cranky/cmds.d/chroot#n180))
> 
> Can we use similar instructions as the ones in https://wiki.ubuntu.com/Kernel/BuildYourOwnKernel to build -intel kernel ?

There is a bunch of things that can go wrong(manual editing/changing file permission), and plenty of edge cases (depending on your host arch or target series/host series).
Almost (or nobody?) one in the kernel team does this process, so I don't want to support someone having issues going down this way.

**Review by bktan8 — APPROVED** [review]
LGTM

**Pull request merged**